code/datalab-app/docker/config/vault/policies/datalab-policy.hcl

```diff
@@ -1,3 +1,3 @@
-path "secret/datalab/*" {
+path "secret/testlab/*" {
```

Comment:
Is this change specific to the environment it is deployed to?

Reply:
Yes, but this is the docker config for the dev environment. The manifests project has a version for each environment